src/theme.rs

```diff
@@ -18,21 +18,49 @@ use crate::piet::Color;
 
 use crate::{Env, Key};
 
-pub const BACKGROUND_COLOR: Key<Color> = Key::new("background_color");
-pub const HOVER_COLOR: Key<Color> = Key::new("hover_color");
-pub const PRESSED_COLOR: Key<Color> = Key::new("pressed_color");
+pub const WINDOW_BACKGROUND_COLOR: Key<Color> = Key::new("window_background_color");
+
 pub const LABEL_COLOR: Key<Color> = Key::new("label_color");
+
+pub const PRIMARY_LIGHT: Key<Color> = Key::new("primary_light");
+pub const PRIMARY_DARK: Key<Color> = Key::new("primary_dark");
+pub const BACKGROUND_LIGHT: Key<Color> = Key::new("background_light");
+pub const BACKGROUND_DARK: Key<Color> = Key::new("background_dark");
+pub const FOREGROUND_LIGHT: Key<Color> = Key::new("foreground_light");
+pub const FOREGROUND_DARK: Key<Color> = Key::new("foreground_dark");
+pub const BUTTON_DARK: Key<Color> = Key::new("button_dark");
+pub const BUTTON_LIGHT: Key<Color> = Key::new("button_light");
+pub const BORDER: Key<Color> = Key::new("border");
+pub const BORDER_LIGHT: Key<Color> = Key::new("border_light");
+pub const SELECTION_COLOR: Key<Color> = Key::new("selection_color");
+pub const CURSOR_COLOR: Key<Color> = Key::new("cursor_color");
+
 pub const FONT_NAME: Key<&str> = Key::new("font_name");
 pub const TEXT_SIZE_NORMAL: Key<f64> = Key::new("text_size_normal");
+//TODO: what's a better name for these?
```

Comment:
Yes, we need better names. I propose `BASIC_WIDGET_HEIGHT` and `BORDERED_WIDGET_HEIGHT`. Definitely open to bikeshedding on these, but want to get a "probably good enough" proposal out there.

Reply:
I went with those names for now. Should I leave in this todo?

Reply:
(I removed it)